README.md

```diff
@@ -59,55 +63,9 @@ Bosh Lite is a lightweight local development environment for BOSH using Warden/G
 
 Well, you know what Cloud Foundry is, otherwise you wouldn't be interested into a service broker to begin with.
 
-* Follow instructions at https://docs.cloudfoundry.org/deploying/cf-release/boshlite/create_a_manifest.html
-* You may like to shorten the polling interval for asynchronous service instance operations in `cf-release/bosh-lite/deployments/cf.yml`:
-```yaml
-properties:
-  cc:
-    broker_client_default_async_poll_interval_seconds: 1
-```
-* Next follow instructions at https://docs.cloudfoundry.org/deploying/cf-release/common/deploy.html to deploy Cloud Foundry
-* If you run into strange errors using bosh <command> and OpenSSL (when generating the manifest), update OpenSSL and patch the scripts:
-```shell
-wget http://www.openssl.org/source/openssl-1.0.1p.tar.gz
-tar -xvzf openssl-1.0.1p.tar.gz
-cd openssl-1.0.1p
-./config --prefix=/usr/
-make
-sudo make install
-# add 2>/dev/null to the bosh <command> calls in ./scripts/generate-bosh-lite-dev-manifest
-```
-* If you run into certificate errors from consul like "Unexpected response code: 500 (rpc error: failed to get conn: x509: certificate has expired or is not yet valid)". This means the certificates has expired and it is required to recreate the certificates. Follow the steps below:
-  * Generate new certificates using the script cf-release/scripts/generate-consul-certs.
-  * Copy the certificates in the relevant section of cf-release/templates/cf-infrastructure-bosh-lite.yml
-  * Regenerate the cf-release deployment manifest
-  * Now recreate the release, upload and deploy it as follows:
-    ```
-    bosh create release && bosh upload release && bosh deploy
-    ```
-* Follow instructions at https://docs.cloudfoundry.org/cf-cli/install-go-cli.html to install cf cli
-
-* If you have installed bosh-lite using [bosh-deployment](https://github.com/cloudfoundry/bosh-deployment), then you can deploy cf using the following commands, using [cf-deployment](https://github.com/cloudfoundry/cf-deployment).
-
-```shell
-git clone https://github.com/cloudfoundry/cf-deployment
-cd cf-deployment
-bosh -e vbox upload-stemcell https://bosh.io/d/stemcells/bosh-warden-boshlite-ubuntu-trusty-go_agent
-bosh -e vbox update-cloud-config iaas-support/bosh-lite/cloud-config.yml
-bosh -e vbox -d cf deploy cf-deployment.yml -o operations/bosh-lite.yml --vars-store deployment-vars.yml -v system_domain=bosh-lite.com
-```
+To run Cloud Foundry with bosh 2.0, instructions from [this](https://starkandwayne.com/blog/running-cloud-foundry-locally-on-bosh-lite-with-bosh2/) article can be followed.
```

Comment:
I would prefer to keep the actual texts, as the blogs may be moved/deleted.

Reply:
Okay. I'll create a wiki page for the same and link it here

Reply:
U dont like the description here? :)

Reply:
better create a .md page and link here?

Reply:
wiki is difficult to maintain version of.